tests/process_monitor/test_process_monitor_handling_communication_from_server.py

```diff
@@ -42,8 +42,6 @@ def test_MantarrayProcessesMonitor__check_and_handle_server_to_main_queue__handl
 ):
     monitor_thread, _, _, _ = test_monitor
 
-    test_process_manager.create_processes()
```

Comment:
What made this line unnecessary in so many tests? Was it not necessary to begin with?